### DISCUSSION TITLE
Improved dashboard badges

### DISCUSSION THREAD

**Discussion by OnlyDrey** [home-assistant/frontend#21303] [Dashboard] [answered]
Hello,

I would like to request updated [Badges](https://www.home-assistant.io/dashboards/badges/) using the configuration of [Mushroom Chips Card](https://github.com/piitaya/lovelace-mushroom/blob/main/docs/cards/chips.md) by @piitaya .

![Old and new badge concept](https://github.com/home-assistant/frontend/assets/22370159/8db24191-c552-4c29-b004-80cc5df36ee0)

The old badges are a bit outdated and it would be great to be able to do more customization from the GUI and not only using YAML. The only addition I would make to the Chips cards is to have the availability to make the Chips scrollable horizontally to be able to remove the "clutter" on the dashboard if needed.

**Comment by KNXBroker** [comment, marked as answer]
Will be part of HA 2024.8
